packages/colorpickers/src/styled/ColorDialog/StyledButton.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'colordialog.button';
```

Comment:
Should all of these styled elements be nested under `colorpickers.*` like other packages?

Reply:
Yeah, updated.